core/frontend/src/tile/map/MapLayerFormatRegistry.ts

```diff
@@ -37,6 +37,9 @@ export interface MapLayerSourceValidation {
 
 /**
  * Options to be passed to the IModelApp on startup which contain access keys for various layer formats
+ * 'BingMaps' must have it's key value set to 'key'
+ * 'MapboxImagery' must have it's key value set to 'access_token'
```

Comment:
what about azure maps?

Reply:
I don't know why but I was expecting this question... according to [Azure Maps documentation](https://docs.microsoft.com/en-us/azure/azure-maps/azure-maps-authentication) it might be `subscription-key`, but I can't test it since I have no clue what is proper base URL to use, nor any valid key.

Reply:
Do you know who added support for Azure Maps? Did they do so without testing it?